win_images/win_packaging.ps1

```diff
@@ -14,14 +14,21 @@ retryInstall git archiver psexec golang mingw StrawberryPerl; Check-Exit
 # Update service is required for dotnet
 Set-Service -Name wuauserv -StartupType "Manual"; Check-Exit
 
-# dotnet is required for wixtoolset
-# Allowing chocolaty to install dotnet breaks in an entirely
-# non-debuggable way.  Workaround this by installing it as
-# a server-feature first.
-Install-WindowsFeature -Name Net-Framework-Core; Check-Exit
-
-# Install wixtoolset for installer build & test.
-retryInstall wixtoolset; Check-Exit
+# Install dotnet as that's the best way to install WiX 4+
+# Choco does not support installing anything over WiX 3.14
+Invoke-WebRequest -Uri https://dotnet.microsoft.com/download/dotnet/scripts/v1/dotnet-install.ps1 -OutFile dotnet-install.ps1
+.\dotnet-install.ps1 -InstallDir 'C:\Program Files\dotnet'
```

Comment:
I would have installed it using `winget` because it automatically adds `dotnet` to the `%PATH%`. Using the script works too, though, and there may be other reasons to choose it that I am not aware of. 

What I have [found ](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script?WT.mc_id=dotnet-35129-website#purpose)is:

```
The intended use of the scripts is for Continuous Integration (CI) scenarios, where:
- The SDK needs to be installed without user interaction and without admin rights.
- The SDK installation doesn't need to persist across multiple CI runs.

The typical sequence of events:
. CI is triggered.
. CI installs the SDK using one of these scripts.
. CI finishes its work and clears temporary data including the SDK installation.

To set up a development environment or to run apps, use the installers rather than these scripts.
```

In the `build_windows.md`[ we are suggesting to install dotnet using winget](https://github.com/containers/podman/pull/22408/files#diff-7208b93342b250997fed5737592fbeec1afdb6271b12ef7e10a12a9200e783c1R80).

Reply:
I pushed a separate commit here to use winget. If this works out, even better. Thanks.

Reply:
Drive-by non-expert comment:  Remember this is preparing a VM image to run CI w/o having to install anything during runtime.  So the "typical sequence" isn't completely correct in this context.  This makes me think that having `%PATH%` all ready to go at CI runtime might be a good idea - it will simplify the CI scripts to which more developers are exposed.

(please take this comment with a grain of salt)